drivers/sensor/bma400/bma400.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_BMA400_BMA400_H_
+#define ZEPHYR_DRIVERS_SENSOR_BMA400_BMA400_H_
+
+#include <device.h>
+#include <sys/util.h>
+#include <zephyr/types.h>
+#include <drivers/sensor.h>
+
+#define BMA400_REG_CHIP_ID          (0x00)
```

Comment:
Why there are parentheses for these single tokens? https://www.kernel.org/doc/html/v4.10/process/coding-style.html#macros-enums-and-rtl